src/Modules/UngradedModules.jl

```diff
@@ -826,12 +826,12 @@ end
     hom(F::FreeMod, G::ModuleFP, V::Vector)
 
 Given a vector `V` of rank `F` elements of `G`, 
-return the homomorphism `F` $\to$ `G` defined by sending `F[i]` to `V[i]`.
+return the homomorphism  defined by sending `F[i]` to `V[i]`.
 """
 hom(F::FreeMod, G::ModuleFP, V::Vector) = FreeModuleHom(F, G, V)
 
 @doc Markdown.doc"""
-    hom(F::FreeMod{T}, G::ModuleFP{T}, A::MatElem{T}) where T
+    hom(F::FreeMod{T}, G::FreeMod{T}, A::MatElem{T}) where T
```

Comment:
May I ask why this change was necessary?

Reply:
@AlexD97 Let us discuss this tomorrow in gathertown or via phone.